packages/civic-auth/reactjs/src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { CivicAuthProvider, UserButton } from "@civic/auth/react";
+
+const App = () => {
+  return (
+      <CivicAuthProvider
+        clientId={"YOUR CLIENT ID"}
+        modalIframe={false}
+      >
+        <UserButton displayMode="redirect"/>
```

Comment:
please remove the displayMode option

Reply:
This gives
`signIn error 
Object
error
: 
Error: iframeRef is required for displayMode 'iframe' at BrowserAuthenticationInitiator.signIn (http://localhost:5173/node_modules/.vite/deps/@civic_auth_react.js?v=5efe2a3e:5479:15)
isPopupError
: 
false
[[Prototype]]
: 
Object`

Reply:
There is another example at the bottom of
https://app.gitbook.com/o/-MbHRfakrvYmMaAu3Awh/s/PrqdylKqzZDnu8FytHLf/integration/react
for iframe mode.